conans/test/unittests/tools/files/test_rename.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pytest
+
+from conan.tools.files import rename
+from conans.test.utils.mocks import ConanFileMock
+from conans.test.utils.tools import TestClient
+
+
+def test_single_patch_file():
```

Comment:
This test is not testing the Windows and the robocopy part. ``os.path.isdir(src)``

Please add a Windows specific test that covers this functionality.